src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/WoodcutterAndroid.java

```diff
@@ -69,13 +69,14 @@ private void breakLog(Block log, Block android, BlockMenu menu, BlockFace face)
 
         if (menu.fits(drop, getOutputSlots())) {
             menu.pushItem(drop, getOutputSlots());
-            log.getWorld().playEffect(log.getLocation(), Effect.STEP_SOUND, log.getType());
+        }
```

Comment:
I think the `fits` check can be completely removed here, `pushItem()` will just not do anything if it cannot be fit. So this may be a bit redundant.